src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Analysis/DistanceMeasurement/DistanceMeasurement.xaml

```diff
@@ -0,0 +1,73 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.UWP.Samples.DistanceMeasurement.DistanceMeasurement"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls"
+    mc:Ignorable="d"
+    d:DesignHeight="300" d:DesignWidth="300">
+    <UserControl.Resources>
+        <Style TargetType="TextBlock">
+            <Setter Property="Margin" Value="0,5,0,5" />
+            <Setter Property="VerticalAlignment" Value="Center"></Setter>
```

Comment:
`<Setter ... />` instead of `<Setter ...></Setter>` to be consistent. #TickyTack